source/_posts/react-firebase.md

```diff
@@ -0,0 +1,891 @@
+---
+title: Build a To-Do Application with React and Firebase
+contributor: Deji Adesoga
+avatar: deji-adesoga.png
+description: Learn how to use build a CRUD application by using React and Firebase 9.

```

Comment:
CRUD or To-Do? :) Let's be consistent, please.

Reply:
@desoga10

Reply:
I'll change the CRUD to To-Do.